### ISSUE TITLE
Prisma go client cannot return many relation

### ISSUE THREAD

**Issue by Disturbing** [prisma/prisma#3456]
**Describe the bug**
I want to get an array (multiple) items from my relation. Go client only generates single return.

**To Reproduce**

Use following schema:
```
type KintoBlock @embedded
{
  blockId: String! @unique
  name: String!
  versionName: String!
  versionType: String!
  buildId: String!
  commitSha: String!
  proxyPort: Int!
  protocol: Protocol!
}

type Deployment @db(name: "deployments")
{
  id: ID! @unique
  createdAt: DateTime! @createdAt
  updatedAt: DateTime! @updatedAt
  deploymentId: String!
  deploymentName: String!
  environmentId: String!
  environmentName: String!
  kintoBlocks: [KintoBlock!]!
}
```
Run `prisma generate`

See Generated code is only returning Client.GetOne, versus GetMany

```
type DeploymentExec struct {
	exec *prisma.Exec
}

func (instance *DeploymentExec) KintoBlocks() *KintoBlockExec {
	ret := instance.exec.Client.**GetOne**(
		instance.exec,
		nil,
		[2]string{"", "KintoBlock"},
		"kintoBlocks",
		[]string{"blockId", "name", "versionName", "versionType", "buildId", "commitSha", "proxyPort"})

	return &KintoBlockExec{ret}
}
```

**Expected behavior**
Should return GetMany when a relation is an array.

**Versions (please complete the following information):**
 - OS: OSX Mojave 10.14 (18A391)
 - prisma/1.22.0-alpha.102 (darwin-x64) node-v10.12.0
 - prismagraphql/prisma:1.21-alpha
 - Mongodb 3.6
 - Golang 1.11

**Additional context**
Looking forward to the fix :) But do let me know if i did something wrong

**Comment by dominikh** [comment]
I cannot reproduce this issue. Given your schema (minus the `protocol` field since you didn't provide its definition), the following code gets generated:

```
func (instance *DeploymentExec) KintoBlocks(params *KintoBlocksParamsExec) *KintoBlockExecArray {
	var wparams *prisma.WhereParams
	if params != nil {
		wparams = &prisma.WhereParams{
			Where:   params.Where,
			OrderBy: (*string)(params.OrderBy),
			Skip:    params.Skip,
			After:   params.After,
			Before:  params.Before,
			First:   params.First,
			Last:    params.Last,
		}
	}

	ret := instance.exec.Client.GetMany(
		instance.exec,
		wparams,
		[3]string{"KintoBlockWhereInput", "KintoBlockOrderByInput", "KintoBlock"},
		"kintoBlocks",
		[]string{"blockId", "name", "versionName", "versionType", "buildId", "commitSha", "proxyPort"})

	return &KintoBlockExecArray{ret}
}
```

**Comment by pantharshit00** [comment]
Closing this now. Please open a new issue if you still somehow facing this 🙂
